ReactSkia/components/RSkComponentText.cpp

```diff
@@ -51,84 +52,135 @@ inline int getNumberOfLines(std::vector<LineMetrics>& metrics, int maxNumberOfLi
             break;
         }
     }
-    if ((minNumberOfLines > 0) && (minNumberOfLines < maxNumberOfLines))
+    if ((minNumberOfLines) && (maxNumberOfLines) && (minNumberOfLines < maxNumberOfLines))
```

Comment:
this function is called only when maxNumberOfLines is set.
Is this check required here again ?

Reply:
maxNumberOfLines check is not required.
Updated